sssd_test_framework/hosts/keycloak.py

```diff
@@ -46,6 +46,28 @@ def __init__(self, *args, **kwargs) -> None:
 
         self.adminpw = self.config.get("adminpw", "Secret123")
 
+        self.client.setdefault("id_provider", "idp")
+        self.client.setdefault("enabled", "true")
+        self.client.setdefault("idp_type", "keycloak:https://master.keycloak.test:8443/auth/admin/realms/master/")
```

Comment:
master.keycloak.test should not be hardcoded. We can use:

```
self.config.setdefault("idp_server", "master.keycloak.test")
```

Reply:
I used `{self.hostname}`, I hope this is what you've meant.